### ISSUE TITLE
[1.3 modules beta1] Sometimes cannot import module

### ISSUE THREAD

**Issue by vjau** [meteor/meteor#5836]
With following folder structure
![folders](https://cloud.githubusercontent.com/assets/5278429/11912511/f54cf762-a63f-11e5-8550-3e345a1329cc.png)

When writing ```import AccountsUIWrapper from "./blaze/login"``` in App.jsx, i get following error 
```
Uncaught Error: Cannot find module './blaze/login'
require @ install.js:77
meteorInstall.app.client.imports.components.App.jsx @ login.jsx:16
fileEvaluate @ install.js:183
require @ install.js:75
(anonymous function) @ app.js?hash=3d1b8876ef190b3c981e64ea65225d9780f932d3:2041
1(anonymous function) @ app.js?hash=3d1b8876ef190b3c981e64ea65225d9780f932d3:20415
```
in browser console.

In login.jsx, i got ```export default AccountsUIWrapper;```

What am i doing wrong ?

**Comment by casperanderseneu** [comment]
@vjau I'm experiencing similar problems in a simple test with a /client/main.jsx that imports from a module file in /client/imports/app.jsx where I wanted to make the simplest react test with import from a module in the same client dir in the app.

however, interestingly enough I made a simple import in a /client/main.js file from a /client/imports/testmodule.js file, and it worked. But when I renamed testmodule.js to testmodule.jsx it returned Uncaught Error:  Cannot find module .... again.

so is this the problem with .jsx files again?

**Comment by nchenbang** [comment]
You've to specify the file extension when importing a .jsx file, which means instead of doing this
```javascript
import AccountsUIWrapper from "./blaze/login"
```
you'll have to do this
```javascript
import AccountsUIWrapper from "./blaze/login.jsx"
```
But I'm not really sure if this is the intended behaviour when importing a .jsx file or this is just another bug?

**Comment by trusktr** [comment]
Adding `.jsx` [doesn't work in my case](https://github.com/meteor/meteor/issues/5864).

**Comment by casperan** [comment]
try
import { blazeToReact } from '../../../utilities/blazeToReact.jsx'

**Comment by trusktr** [comment]
This might be the problem: I discovered that after updating package.json dependencies and running `npm install`, that package won't be available until after I run `meteor reset` and then restart the server. Without resetting I get the `Uncaught Error: Cannot find module` for the newly installed package. (@benjamn)

**Comment by vjau** [comment]
Yes @trusktr, it seems to work after a ```meteor reset```

**Comment by benjamn** [comment]
I can confirm this is a bug that I believe will be fixed by https://github.com/meteor/meteor/issues/5815.

**Comment by avital** [comment]
Should be fixed in `1.3-modules-beta.3`
